airflow/load_data_forecast.py

```diff
@@ -0,0 +1,102 @@
+# These args will get passed on to each operator
+import json 
+import pandas as pd
+import numpy as np
+# Needed for load_data
+from corona_ts.data_utils.data_crawler import load_data
+
+def add_gcp_connection(**kwargs):
+    new_conn = Connection(
+            conn_id="google_cloud_default",
+            conn_type='google_cloud_platform',
+    )
+    extra_field = {
+        "extra__google_cloud_platform__scope": "https://www.googleapis.com/auth/cloud-platform",
+        "extra__google_cloud_platform__project": "task-ts",
+        "extra__google_cloud_platform__key_path": '/home/efawe/airflow/dags/task-ts-562f84386e61.json'
+    }
+
+    session = settings.Session()
+
+    #checking if connection exist
+    if session.query(Connection).filter(Connection.conn_id == new_conn.conn_id).first():
+        my_connection = session.query(Connection).filter(Connection.conn_id == new_conn.conn_id).one()
+        my_connection.set_extra(json.dumps(extra_field))
+        session.add(my_connection)
+        session.commit()
+    else: #if it doesn't exit create one
+        new_conn.set_extra(json.dumps(extra_field))
+        session.add(new_conn)
+        session.commit()
+
+def dataToGCS(csv_name: str, folder_name: str,
+                   bucket_name="task_ts_data", **kwargs):
+    hook = GoogleCloudStorageHook()
+    data = load_data()
+    df = pd.DataFrame(data=data)
+    df.to_csv('mobility_data.csv', index=False)
+
+    hook.upload(bucket_name, 
+                object='{}/{}.csv'.format(folder_name, csv_name), 
+                filename='mobility_data.csv', 
+                mime_type='text/csv')
+
+default_args = {
+    'owner': 'airflow',
+    'depends_on_past': False,
+    'start_date': days_ago(2),
+    'email': ['airflow@example.com'],
+    'email_on_failure': False,
+    'email_on_retry': False,
+    'retries': 1,
+    'retry_delay': timedelta(minutes=5),
+    # 'queue': 'bash_queue',
+    # 'pool': 'backfill',
+    # 'priority_weight': 10,
+    # 'end_date': datetime(2016, 1, 1),
+    # 'wait_for_downstream': False,
+    # 'dag': dag,
+    # 'sla': timedelta(hours=2),
+    # 'execution_timeout': timedelta(seconds=300),
+    # 'on_failure_callback': some_function,
+    # 'on_success_callback': some_other_function,
+    # 'on_retry_callback': another_function,
+    # 'sla_miss_callback': yet_another_function,
+    # 'trigger_rule': 'all_success'
+}
+dag = DAG(
+    'load_data_forecast',
+    default_args=default_args,
+    description='DAG to populate mobility data for forecast team',
+    schedule_interval=timedelta(days=1),
+)
+
+activateGCP = PythonOperator(
+        task_id='add_gcp_connection_python',
+        python_callable=add_gcp_connection,
+        provide_context=True,
+        dag = dag,
+    )
+
+dataToGCS_task = PythonOperator(
+        task_id='simpleNumpyToGCS',
+        python_callable=dataToGCS,
+        provide_context=True,
+        op_kwargs={'csv_name': 'mobility_data', 'folder_name': 'airflow'},
```

Comment:
Where the DAG writes to should be the date `2019-09-10`

Reply:
Are you implying that it should just be a static folder or as data is pulled it would populate more folders with increasing dates